apps/user-group-cdc/src/main/java/it/pagopa/selfcare/user/event/entity/UserGroupEntity.java

```diff
@@ -4,18 +4,23 @@
 import io.quarkus.mongodb.panache.reactive.ReactivePanacheMongoEntity;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
 import lombok.experimental.FieldNameConstants;
+import org.bson.types.ObjectId;
 
 import java.time.Instant;
 import java.util.Set;
 
 @EqualsAndHashCode(callSuper = true)
 @Data
+@Getter
+@Setter
```

Comment:
You're using both @Data, @Getter, and @Setter. Since @Data already includes @Getter and @Setter, the additional annotations are redundant....
you could Remove @Getter and @Setter to avoid redundancy